docs/docs/configuration/license_plate_recognition.md

```diff
@@ -22,7 +23,7 @@ Users without a model that detects license plates can still run LPR. Frigate use
 
 :::note
 
-Frigate needs to first detect a `car` before it can recognize a license plate. If you're using a dedicated LPR camera or have a zoomed-in view, make sure the camera captures enough of the `car` for Frigate to detect it reliably.
+In the default mode, Frigate's LPR needs to first detect a `car` before it can recognize a license plate. If you're using a dedicated LPR camera and have a zoomed-in view where a `car` will not be detected, you can still run LPR, but the configuration parameters will differ from the default mode. See the Dedicated LPR Camera section below.
```

Comment:
Maybe add a hyperlink to the last sentence

Reply:
Good call